sphinxcontrib/websupport/builder.py

```diff
@@ -178,6 +178,25 @@ def handle_finish(self):
                     shutil.rmtree(dst)
                 shutil.move(src, dst)
 
+        # copy resource files to static dir
+        resources = [
+            'ajax-loader.gif',
+            'comment-bright.png',
+            'comment-close.png',
+            'comment.png',
+            'down-pressed.png',
+            'down.png',
+            'up-pressed.png',
+            'up.png',
+        ]
+        package_dir = path.abspath(path.dirname(__file__))
```

Comment:
I think package_dir would be commonly used. How about moving this to module global?

Reply:
I fixed it.